src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -17,28 +17,29 @@ const generateInitialOptions = series => {
     tooltips: {
       callbacks: {
         footer: (tooltipItems, data) => {
-          const tooltipData = [];
-          let delta = 'n/a';
-          let deltaPercentage = 'n/a';
-          let dataset = 'n/a';
-          let currentData = 'n/a';
+          // get data from all points of selected series
+          const dataset = data.datasets[tooltipItems[0].datasetIndex].data;
+          // get data from selected point
+          const currentData = dataset[tooltipItems[0].index].y;
+          const tooltipData = []; // footer's text lines will be stored here
+          let delta = 0;
+          let deltaPercentage = 0;
```

Comment:
It makes it easier to review what's changed if the order of the existing variables stays the same (unless there was a reason to change it).

Reply:
not an important reason: `dataset`and `currrentData`are now 'constants' so I thought they should go at the top along with `tooltip` which is constant as well, otherwise I would end up having the next lines `const` -> `let`-> `let`->`const`->`const`.
Should I change the order then?

Reply:
I'm not sure the ordering of `const` and `let` matters - it makes it easier to read the diff to not reorder. But it's moot at this point so you can leave it.